app/src/index.css

```diff
@@ -123,6 +123,10 @@
 
         --input: 12 6.5% 15.1%; /* shadcn */
         --ring: 24 5.7% 82.9%; /* shadcn */
+
+        img[src*='zed.svg'] {
```

Comment:
Is there a way to scope this filter into the SVG so that it's not in our global styles @GooglyBlox ?